utilities/backup/backup_engine_test.cc

```diff
@@ -657,7 +657,7 @@ class BackupEngineTest : public testing::Test {
     engine_options_->max_background_operations = 7;
 
     // delete old files in db
-    DestroyDB(dbname_, options_);
+    DestoryDBWithoutCheck(dbname_, options_);
```

Comment:
normally I ignore spelling, but since it's many occurrences, maybe Destory -> Destroy